### PR TITLE
Added some minor changes to support Swift 3.0

### PR DESCRIPTION
**Pull request by garethpaul**
Added minor changes to support Swift 3.0. It may likely be best to wait on a non beta release of Xcode, before considering a merge of this PR.

### CODE REVIEW

**Comment by facebook-github-bot-1** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

**Comment by facebook-github-bot-2** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

**Comment by nscoding** [comment]
I don't usually merge changes for swift until it's officially released including the newer version of Xcode, I could make a branch for this though.

**Comment by nscoding** [comment]
There are also some build errors in this case.

**Comment by nscoding** [comment]
I pushed another branch https://github.com/facebook/ios-snapshot-test-case/tree/swift-beta-3 where you can open a PR against it, once Swift 3 is released I will then merge it into master.

File: FBSnapshotTestCase/SwiftSupport.swift [hunk 1]
```diff
@@ -9,33 +9,33 @@
...
+  private func FBSnapshotVerifyViewOrLayer(_ viewOrLayer: AnyObject, identifier: String = "", suffixes: OrderedSet = FBSnapshotTestCaseDefaultSuffixes(), tolerance: CGFloat = 0, file: StaticString = #file, line: UInt = #line) {
+    let envReferenceImageDirectory = self.getReferenceImageDirectory(withDefault: FB_REFERENCE_IMAGE_DIR)
     var error: NSError?
     var comparisonSuccess = false
 
     if let envReferenceImageDirectory = envReferenceImageDirectory {
       for suffix in suffixes {
         let referenceImagesDirectory = "\(envReferenceImageDirectory)\(suffix)"
-        if viewOrLayer.isKindOfClass(UIView) {
+        if viewOrLayer.isKind(UIView) {
```
Comment (Grubas7) [inline FBSnapshotTestCase/SwiftSupport.swift]: It's not a very Swifty. I would recommend:
`if let view = viewOrLayer as? UIView {`
Same for `CALayer`

**Comment by nscoding** [comment]
Merged #189 in master

**Pull request closed without merging**